### ISSUE TITLE
Powershell ISE exposes the full .net to the console session?

### ISSUE THREAD

**Issue by read-0nly** [PowerShell/PowerShell#9584]
# Steps to reproduce

```This was tested on Powershell 5.1 - I realize .net core 2.2 now includes the bitmap class. I suspect the overarching behavior likely still reproduces but am not in a position to confirm with Powershell 6 at the moment

```To test with Powershell 5.1, create any powershell script that references System.Drawing.Bitmap

```Run in ISE and watch it succeed

```Run in console and it fails because System.Drawing.Bitmap isn't part of .net core per https://github.com/PowerShell/PowerShell/issues/5704

```

# Expected behavior
```For the behavior to be consistent between IDE and Console

```

# Actual behavior

```Some classes seem to be available to the ISE that aren't exposed to the vanilla console

```As far as I can tell, this is because the ISE exposes the full .net library whereas the console loads only .net core as expected. I suspect the inline console in the ISE inherits the libraries loaded by the ISE and the ISE was built for the full .net library, not just core.

```

# Environment data

```WMF 5.1, .net core 2. Not at home, will provide full output later.

```

**Comment by read-0nly** [comment]
I can't figure out how I broke the template, closing and will reopen once I've tested with the latest version of WMF. This isn't so much a glaring issue as unexpected behavior, and you have to be doing weird things powershell wasn't really meant for to run into it.

For an example of this behavior, https://github.com/read-0nly/PowerDMM

**Comment by fMichaleczek** [comment]
@read-0nly This repository is for PowerShell Core and your issue is for Windows PowerShell.
Move your issue to your repository, i will have a look

**Comment by chuanjiao10** [comment]
hi @vexx32  :
"System.Drawing.Common" , Can it work on linux?

**Comment by vexx32** [comment]
I believe that System.Drawing will work via System.Drawing.Common on unix systems as it runs via Mono assemblies.

_However_, System.Drawing is highly **unstable** on unix. @TylerLeonhardt can attest to that as I had him test PSWordCloud on Mac and Linux while it was still using System.Drawing assemblies. We had very poor results, a whole lot of errors. Some smaller applications work, but it appears when you try to instantiate true graphical surfaces a lot can come crashing down very quickly.

That was what pushed me to rewrite PSWordCloud with SkiaSharp instead, really.

**Comment by chuanjiao10** [comment]
en,I am looking forward to call System.Windows.Forms on .net core 3 linux.
i want try it on ps7 preview.

**Comment by read-0nly** [comment]
Oh man I am so many levels of stupid. Yep, this is powershell core and I'm in the wrong place. Yep, if I add-type first it works. I had tried add-type, but was trying to add system.drawing.bitmap. When I found the issue page and compared the .net core versions it seemed to hold up with what I was seeing, and tunnel-visioned on that. So much time wasted trying to figure this out.

Well, I've seen it said that the best way to get help with debugging is to be wrong loudly and with confidence and let people tell you why you're wrong. I guess it holds water. Thanks for helping this idiot along lol
